notesserver/settings/yaml_config.py

```diff
@@ -42,7 +42,7 @@
     DATABASES['default'][override] = value
 
 if ES_DISABLED:
-    HAYSTACK_CONNECTIONS = {}
-    INSTALLED_APPS.remove('haystack')
+    ELASTICSEARCH_DSL = {}
+    INSTALLED_APPS = [i for i in INSTALLED_APPS if i not in ES_APPS]
```

Comment:
Don't we only add `ES_APPS` to `INSTALLED_APPS` if `not ES_DISABLED` anyway? In which case we wouldn't need this line, right? (Same question for other instance above in `test_es_disabled.py`)

Reply:
Oh yes, you're right Michael. This line doesn't make any sense. Thanks for noticing this! 
I'll remove both here and in test_es_disabled.py